Dockerfile

```diff
@@ -11,10 +11,12 @@
 #  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #  License for the specific language governing permissions and limitations under
 #  the License.
+
```

Comment:
Placing this comment here but it is more generic to the PR - could you please add a bit more detail regarding the need for this fix (OSU's setup has the OBU generate BSMs, which are forwarded to the driver, which is different from the typical setup used for CARMA Platform testing). Additionally, can we change the PR name to more closely reflect the fix that has been implemented?

Finally, for other changes/updates in the PR that are not relevant to the primary fix, could you please include a note in the PR description for them? I believe these are [1] the new devcontainer support (if they remain in the PR) and [2] the PSM psid fix.

Reply:
@JonSmet Updated!